index.js

```diff
@@ -14,6 +14,10 @@ function defaultKey (req, file, cb) {
   })
 }
 
+function defaultAcl (req, file, cb) {
+  setImmediate(function () { cb(null, 'private') })
```

Comment:
Might as well fix it whilst we're here. I've made this change in the newest commits now.